randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -209,7 +224,13 @@ Extra - asset_id: 4
   > Door to Bowling Alley
       Trivial
   > Pickup (Topmost Missile Expansion)
-      Hi-Jump Boots or Space Jump
+      Any of the following:
+          Hi-Jump Boots or Space Jump or Knowledge (Intermediate)
```

Comment:
Why knowledge? Those are always nice to have a comment

Reply:
Knowledge that you can lead the Trippers over to a platform on the far right by jumpting to keep them on screen and get onto them without any movement tech at all. I think most people assume you need ice or highjump.

Reply:
> Those are always nice to have a comment

:)

Same goes for the one in seafloor